lib/board.rb

```diff
@@ -0,0 +1,51 @@
+require 'pry'
+
+class Board
+  attr_accessor :cells
+
+  def initialize
+    reset!
+  end
+
+  def reset!
+    @cells = [" "," "," "," "," "," "," "," "," "]
+  end
+
+  def display
+    puts " #{cells[0]} | #{cells[1]} | #{cells[2]} "
+    puts "-----------"
+    puts " #{cells[3]} | #{cells[4]} | #{cells[5]} "
+    puts "-----------"
+    puts " #{cells[6]} | #{cells[7]} | #{cells[8]} "
+  end
+
+  def position(input)
+    self.cells[(input.to_i)-1]
+  end
+  
+  def full?
+    if cells.detect {|cell| cell == " " }; false
```

Comment:
Avoid returning boolean literals from `if` statements, relying instead on the implicit return value of the boolean expression. You can reverse (or negate) the boolean with the `!` (bang) operator.
```
def full?
  !cells.detect {|cell| cell == " " }
end
```